libcudacxx/include/cuda/std/__type_traits/common_reference.h

```diff
@@ -76,13 +76,9 @@ using __cond_res = decltype(false ? _CUDA_VSTD::declval<_Xp (&)()>()() : _CUDA_V
 // Let `XREF(A)` denote a unary alias template `T` such that `T<U>` denotes the same type as `U`
 // with the addition of `A`'s cv and reference qualifiers, for a non-reference cv-unqualified type
 // `U`.
-// [Note: `XREF(A)` is `__xref<A>::template __apply`]
+// [Note: `XREF(A)` is `__xref<A>::template __call`]
 template <class _Tp>
-struct __xref
-{
-  template <class _Up>
-  using __apply = __copy_cvref_t<_Tp, _Up>;
-};
+using __xref = __apply_cv_fn<_Tp>;
```

Comment:
I am a bit troubled hat this does not name `__apply_cvref_fn` because we should explicitly also copy reference qualifiers

Reply:
Good catch